index.html

```diff
@@ -11,105 +11,105 @@
+      <p class="blockchain-padding white-color">Learn from tutors from various countires on how to develop Blochchiain Platforms</p>
      </div>
-     
-      <div class="main-grid-item">
-        <div class="crpyto-icon"></div>
-        <h3 class="red-color">Crypto Analysis</h3>
-        <p class="white-color">Learn from tutors from various countires on how to develop effectively analyse the crypto market improved trading strategy</p>
-      </div>
 
-      
       <div class="main-grid-item">
         <div class="editing-icon"></div>
         <h3 class="red-color">Editing</h3>
-        <p class="white-color">Learn from tutors from various countires on how to edit your publications</p>
+        <p class="editing-padding white-color">Learn from tutors from various countires on how to edit your publications</p>
       </div>
-      <button class="register-button red-background" type="button">Register for Next Cohorts</button>       
-   </section> 
-  
-    <section class="featured-speakers">
-    <h2 class="align-center">Featured Tutors</h2>
-    <div class="flex-container display-flex">
-      <div class="chinedu-image"></div>
-      <div><h4>Chinedu Ezea</h4>
-      <p>Chinedu is a doctor of philosophy at Nnamdi Azikiwe University</p>
-      <p>He is a mathematician and a professional editors with several publications in high impact factor journals</p>
-    </div>
-  </div>
-  </section>
+      </div>
+      <div class="align-center">
+      <button class="mobile-btn register-button red-background" type="button">Register for Next Cohorts</button>       
+      <a class="regiter-link white" href="">Register for Next Cohorts</a>
+    </div>    
+    </section>  
+    <section id="tutor">
+    <h2 class="align-center">Featured Tutors</h2><hr class="red-background tutor-dash">
+    <div id="featured-speakers-id"></div> 
+</section>
+<div class="align-center">
+<button class="more" type="button">More</button>
+</div>
   </main>
-  <footer class="footer">
-    <div class="dark-background partners-container">
-    <h5 class="align-center">Partners</h5>
+  <footer class="footer-index">
+    <div class="dark-background align-center partners-container">
+    <h5 class="partner white-color">Partners</h5><hr class="red-background tutor-dash">
     <ul class="flex-conatiner display-flex">
-      <li class="flex-item"><a href="">ABCDE</a></li>
+      <li class="flex-item"><a class="partner-1" href="">ABCDE</a></li>
       <li class="flex-item"><a href="">FGHIJ</a></li>
-      <li class="flex-item"><a href="">KLMNO</a></li>
-      <li class="flex-item"><a href="">QRSTU</a></li>
-      <li class="flex-item"><a href="">PQRST</a></li>
+      <li class="flex-item"><a class="partner-1" href="">KLMNO</a></li>
+      <li class="flex-item"><a href="">PQRSTABCSEF</a></li>
       <li class="flex-item"><a href="">UVWXY</a></li>         
     </ul> 
   </div>
   <div class="copyright-flex-container display-flex">
     <div> &copy;SkillHub<br>Academy</div>    
```

Comment:
- [ ] [OPTIONAL] You could use an image for the logo so your design is similar to the provided one

Reply:
Done!